src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -107,3 +107,5 @@ fun <T, R> List<Observable<T>>.zip(zipFunction: (args: List<T>) -> R): Observabl
  * Returns an Observable that emits the items emitted by the source Observable, converted to the specified type.
  */
 inline fun <reified R : Any> Observable<*>.cast(): Observable<R> = cast(R::class.java)
+
+fun<T> observableDefer(x: T) = Observable.defer { Observable.just(x) }
```

Comment:
This is what `just` does on its own, why defer a practically constant source? Besides, `just` allows nice optimizations in `merge`, `flatMap`, `concatMap`, `switchMap` etc.